packages/@okta/vuepress-site/docs/guides/social-login/main/line/noemail.md

```diff
@@ -0,0 +1,52 @@
+## Handle users without email addresses
+
+### The user has an email address in LINE, but it's not in Okta
+
+Before you proceed to the next section, test your integration with a user who has an email in LINE. If you don't see their email in Okta, verify that you have [email address permission](https://developers.line.biz/en/docs/line-login/integrate-line-login/#applying-for-email-permission) in your LINE channel. If you've confirmed that your channel has email permission and you're still unable to see user emails in Okta, [contact support](https://support.okta.com/help/s/opencase).
```

Comment:
The link text reads awkward to me. I know it's the content in the URL, but I wonder what "email address permission" is. The header content in the Link docs is more clear, imo.
```suggestion
Before you proceed to the next section, test your integration with a user who has an email in LINE. If you don't see their email in Okta, verify that you have permission to [access the user's email address](https://developers.line.biz/en/docs/line-login/integrate-line-login/#applying-for-email-permission) in your LINE channel. If you've confirmed that your channel has email permission and you're still unable to see user emails in Okta, [contact support](https://support.okta.com/help/s/opencase).
```